integration/gateway/gateway_suite_test.go

```diff
@@ -94,3 +103,50 @@ func chaincodeArgs(transactionName string, args ...[]byte) [][]byte {
 
 	return result
 }
+
+func NewECDSAPrivateKey() *ecdsa.PrivateKey {
```

Comment:
Instead of writing this boiler plate code, my suggestion would be to simply use a built-in user that would not be authorized -- you can use "orderer admin" in this case. 
`network.OrdererUserSigner(network.Orderer("orderer"), "Admin")`